views/layout.html

```diff
@@ -18,6 +18,7 @@
 {% from "govuk/components/textarea/macro.njk"      import govukTextarea %}
 {% from "govuk/components/file-upload/macro.njk"   import govukFileUpload %}
 {% from "govuk/components/inset-text/macro.njk"    import govukInsetText %}
+{% from "govuk/components/warning-text/macro.njk"  import govukWarningText %}
```

Comment:
is this needed? I don't see any nunjucks changes using this in this pull request.

Reply:
confirmation-still-required.html  on line 52

Reply:
It should have gone on with ticket LFA 1364